client/pages/courses/index.vue

```diff
@@ -373,13 +376,19 @@ export default {
             return this.$store.getters['planner/isInPlan'](this.courseInfo)
         },
         categories_results() {
-            const [availableFilter, campusFilter, subjectFilter, courseFilter, dayFilter, profFilter, attrFilter] = this.selected,
+            //TODO actual filtering
+            const [availableFilter, scheduleFilter, campusFilter, subjectFilter, courseFilter, dayFilter, profFilter, attrFilter] = this.selected,
                 filter = course => {
                     if (availableFilter === Boolean(availableFilter)) {
                         if (course.available !== availableFilter) {
                             return false
                         }
                     }
+                    if (!scheduleFilter) {
```

Comment:
This logic doesn't flow like the others could we make it so that it reads `if(scheduleFilter) {`